packages/app/src/layouts/default.vue

```diff
@@ -119,11 +120,23 @@ mutation MainApp_ResetErrorsAndLoadConfig($id: ID!) {
 
 const currentRoute = useRoute()
 
+const headedInitialLoad = ref(false)
+
+// Do not render the banner until the header has finished loading.
+// This helps to make the UI stable and prevent elements from jumping around.
+function handleHeaderLoaded () {
+  headedInitialLoad.value = true
+}
+
 const showHeader = computed(() => {
   return currentRoute.meta.header !== false
 })
 
 const showEnableNotificationsBanner = computed(() => {
+  if (!showHeader.value || !headedInitialLoad.value) {
+    return false
+  }
```

Comment:
This conditional is not needed.  The fix for the `pause` variable in the query is what solved the bug.  When the app would refresh on the `runner` when `showHeader` would be false, then the query would never run unless the page was refreshed on a page other than the `runner`.  No need to couple the logic for `showEnableNotificationsBanner` to the header query here.

Reply:
Yes you are correct, good point, updated.